app/views/users/registrations/edit.html.erb

```diff
@@ -50,4 +50,4 @@
 
 <p>Unhappy? <%= button_to "Delete my account", registration_path(resource_name), method: :delete %></p>
 
-<%= link_to "Back", :back %>
+<%= link_to "Back", root_path %>
```

Comment:
User doesn't have access to this view anymore so can ignore this (using custom edit view now not the devise view)